docs/flag_source_configuration.md

```diff
@@ -1,7 +1,96 @@
 # Flag Source configuration
 
 The injected sidecar is configured using the `FlagSourceConfiguration` custom resource definition. 
-The `openfeature.dev/flagsourceconfiguration` annotation is used to assign Pods with their respective`FlagSourceConfiguration` CRs.
+The `openfeature.dev/flagsourceconfiguration` annotation is used to assign Pods with their respective`FlagSourceConfiguration` custom resources.
+
+A minimal example of a `FlagSourceConfiguration` is given below,
+
+```yaml
+apiVersion: core.openfeature.dev/v1alpha3
+kind: FlagSourceConfiguration
+metadata:
+  name: flag-source-configuration
+spec:
+  sources:                        # flag sources for the injected flagd
+    - source: flags/sample-flags  # FlagSourceConfiguration - namespace/name
+      provider: kubernetes        # kubernetes flag source backed by FlagSourceConfiguration custom resource
+  port: 8080                      # port of the flagd sidecar
+```
+
+## Feature flag sources
+
+This section explains how to configure feature flag sources to injected flag sidecar.
+
+`FlagSourceConfiguration` support multiple flag sources. Sources are configured as a list and given below are supported sources and their configurations,
+
+### kubernetes aka `FeatureFlagConfiguration`
```

Comment:
both this, and `flagd-proxy` make use of `FeatureFlagConfiguration`. I think you might want to put these two closer together, and conceptually link and contrast them (probably just highlight that they both use `FeatureFlagConfiguration`, but one uses a proxy and doesn't require direct access of the workload to k8s).

Reply:
Good point. Added a description and reordered the source list to keep k8s & proxy approach close to each other